open-media-match/src/OpenMediaMatch/storage/default.py

```diff
@@ -113,21 +113,23 @@ def _get_bank(self, name: str) -> t.Optional[database.Bank]:
             select(database.Bank).where(database.Bank.name == name)
         ).scalar_one_or_none()
 
-    def bank_update(self, bank: BankConfig, *, create: bool = False) -> None:
+    def bank_update(
+        self, bank: BankConfig, *, create: bool = False, name: t.Optional[str] = None
+    ) -> None:
         if create:
             database.db.session.add(database.Bank.from_storage_iface_cls(bank))
         else:
-            database.db.session.execute(
-                update(database.Bank)
-                .where(database.Bank.name == bank.name)
-                .values(enabled_ratio=bank.matching_enabled_ratio)
-            )
+            previous = database.Bank.query.filter_by(name=name).one_or_404()
+            previous.name = bank.name
+            previous.enabled_ratio = 1 if bank.enabled else 0
```

Comment:
blocking: Shouldn't this be be 

```
previous.enabled_ratio = bank.enabled_ratio
```
?

Reply:
actually it can be previous.enabled_ration  = bank.matching_enabled_ratio I think